frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/review-and-create-step.tsx

```diff
@@ -51,14 +55,19 @@ export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({
         <ReviewListBody noValue={!zones.size}>
           <p>{pluralize(zones.size, 'zone')}</p>
         </ReviewListBody>
-        <ReviewListTitle text="Configure" />
         {(encryption.clusterWide || encryption.storageClass) && (
-          <ReviewListBody>
-            <p className="ocs-install-wizard__review-encryption">Enable Encryption</p>
-            {encryption.advanced && kms.hasHandled && (
-              <p>Connected to external key management service: {kms.name}</p>
-            )}
-          </ReviewListBody>
+          <>
+            <ReviewListTitle text="Configure" />
+            <ReviewListBody>
+              <p className="ocs-install-wizard__review-encryption">Enable Encryption</p>
+              {encryption.advanced && kms.hasHandled && (
+                <p className="ocs-install-wizard__review-encryption">
+                  Connect to external key management service: {kms.name}
```

Comment:
```suggestion
                  External key management service: {kms.name}
```

Reply:
As per UX, its `Connected` since its connected after `Create` button click, so changed it to `Connect`.